src/main/java/mops/businesslogic/FileInfoServiceImpl.java

```diff
@@ -0,0 +1,67 @@
+package mops.businesslogic;
+
+import lombok.AllArgsConstructor;
+import mops.exception.MopsException;
+import mops.persistence.FileInfoRepository;
+import mops.persistence.file.FileInfo;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@AllArgsConstructor
+public class FileInfoServiceImpl implements FileInfoService {
+
+    /**
+     * reference to fileInfoRepository.
+     */
+    private final FileInfoRepository fileInfoRepo;
+
+    /**
+     * @param dirId directory id
+     * @return a list of files in that directory
+     */
```

Comment:
you could use
```suggestion
    /**
     * {@inheritDoc}
     */
```
same for the others. This keeps the java doc update and there will be no conflicts with the interface.